src/gxhash/mod.rs

```diff
@@ -116,15 +115,19 @@ pub(crate) unsafe fn compress_all(input: &[u8]) -> State {
 }
 
 #[inline(always)]
-unsafe fn compress_many(mut ptr: *const State, hash_vector: State, remaining_bytes: usize) -> State {
-
+unsafe fn compress_many(
+    mut ptr: *const State,
+    hash_vector: State,
+    remaining_bytes: usize,
+) -> State {
```

Comment:
This "maximum line length" is [a convention](https://rustc-dev-guide.rust-lang.org/conventions.html#line-length) that doesn't make a consensus. On my end, I think the limit of 80 or 100 is way too short, thus I'd like to keep it on a single line in this case

Reply:
Many crates now expect authors of PRs to run `clippy` and `rustfmt` before submitting so I made that a habit.

> This "maximum line length" is [a convention](https://rustc-dev-guide.rust-lang.org/conventions.html#line-length) that doesn't make a consensus.

Understood.
Shall we add a `rustfmt.toml` with your line length preference?
I think this is best. Particularly when people have PRs with actual blocks of code (vs my meager one liners in this one).

They can then run `rustfmt` once, before submitting, and get everything laid out in a way you prefer.

Reply:
Good idea! I'll do a basic rustfmt

Reply:
I played around with rustfmt and while it has undeniable benefits for collaboration I am not ready with such strictness (eg I agree with most of what is said by **kornel** [here](https://internals.rust-lang.org/t/forced-rustfmt-is-a-roadblock-to-contributing/11913)).  
One example is [this line](https://github.com/ogxd/gxhash/blob/c6c06323f3845defa7c189d795ba77f5a9614f6f/src/gxhash/platform/x86_256.rs#L45) that rustfmt forces to break. I would be fine with breaking the enumeration into two parts of 16 bytes each (one part by line) but rustfmt just doesn't care and enforces a formatting that does not make sense in the context.   

I have just committed on main an #[rustfmt::skip] so that future contributors won't struggle with automated code style changes on this repository.  

Are you ok if I amend your commit to remove code style changes and merge it?

Reply:
Absolutely!